tests/test_enterprise_kafka.py

```diff
@@ -51,7 +52,6 @@ def test_adb_config(self):
                 advertised.listeners=PLAINTEXT://adb-metrics:9092
                 broker.id=1
                 confluent.metrics.reporter.bootstrap.servers=adb-metrics:9092
-                confluent.metrics.reporter.publish.ms=1000
```

Comment:
Is this to reduce load?

Reply:
This is the expected list of properties that the actual config is compared against. It wasn't set in the actual list (I dont know why).